components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,86 @@
+@import "@not-govuk/sass-base";
+@import "govuk-frontend/govuk/helpers/_colour.scss";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  p {
+    text-decoration: inherit;
+    font-size: inherit;
+  }
+
+  &__item {
+    @include govuk-responsive-padding(2, "bottom");
+    @include govuk-responsive-padding(5, "left");
+    /* padding size 6 looks better with smaller viewports. Can media queries
+    be used here? */
```

Comment:
This comment should go.

Reply:
Media queries happen at the outer level, you just need to create a new rule with the media query and define the different padding size in there.

Reply:
But also I don't think we should manually use media queries if we can help it.